### PR TITLE
[android] adaptive height for hotels filter dialog

### PR DESCRIPTION
**Pull request by goblinr**
Adaptive height for hotels filter dialog(https://jira.mail.ru/browse/MAPSME-3701).

@alexzatsepin 
@mpimenov 
PTAL

### CODE REVIEW

**Comment by mpimenov** [comment]
LGTM

File: android/src/com/mapswithme/maps/search/HotelsFilterView.java [hunk 1]
```diff
@@ -109,6 +123,29 @@ public void onClick(View v)
...
+      ViewGroup.LayoutParams lp = mFrame.getLayoutParams();
+      lp.height = height;
+      mFrame.setLayoutParams(lp);
+    }
+    else
+    {
+      ViewGroup.LayoutParams lp = mFrame.getLayoutParams();
+      lp.height = LayoutParams.WRAP_CONTENT;
+      mFrame.setLayoutParams(lp);
+    }
```
Comment (mpimenov) [inline android/src/com/mapswithme/maps/search/HotelsFilterView.java]:
```
  if (height >= getMeasuredHeight())
    LayoutParams.WRAP_CONTENT;
    
  ViewGroup.LayoutParams lp = mFrame.getLayoutParams();
  lp.height = height;
  mFrame.setLayoutParams(lp);
```

Comment (goblinr) [inline android/src/com/mapswithme/maps/search/HotelsFilterView.java]: Done

**Comment by goblinr** [comment]
@mpimenov PTAL

**Pull request merged**